crates/iota-core/src/unit_tests/authority_tests.rs

```diff
@@ -4713,6 +4713,7 @@ async fn test_shared_object_transaction_ok() {
         .epoch_store_for_testing()
         .get_shared_locks(&certificate.key())
         .expect("Reading shared locks should not fail")
+        .expect("Locks should be set")
```

Comment:
These comments are in the wrong format. They should describe the error, not what should occur. More below and also some are incorrectly capitalized

Reply:
Sorry I am not following what format is wrong? Can you give me an example?

Reply:
1. These messages should be all lowercase
2. The phrasing should be "locks are not set" instead of "locks should be set", I.E. Descriptive not prescriptive

Reply:
Fixed at [d206cd8](https://github.com/iotaledger/iota/pull/4757/commits/d206cd846c0a10ff1496e207cf2a504492d43e66).